### ISSUE TITLE
OCSP Stapling check causes non-zero exit code when --phone-out not set

### ISSUE THREAD

**Issue by andrewbonney** [drwetter/testssl.sh#1223]
I'm not certain if this is a bug or simply an error in my setup, but just in case it's the former...

When testing using `testssl.sh -S` and with `--phone-out` not set because I'm aware my OCSP server is unreachable, I was expecting the OCSP stapling check to be skipped around https://github.com/drwetter/testssl.sh/blob/2.9dev/testssl.sh#L8500. Instead, the `$ocsp_response` variable was set to the following:

```
OCSP response:
===============================================================================
Responder Error: trylater (3)
```

Which resulted in the tests hitting the following error handler https://github.com/drwetter/testssl.sh/blob/2.9dev/testssl.sh#L8520

Linux 4.4.0-142-generic #168-Ubuntu SMP Wed Jan 16 21:00:45 UTC 2019 x86_64 x86_64 x86_64 GNU/Linux
testssl.sh       3.0rc4 from https://testssl.sh/dev/
(4fd69fc 2019-04-01 11:49:17 -- )
OpenSSL 1.0.2-chacha (1.0.2k-dev)

**Comment by drwetter** [comment]
If the IP can be reached from the outside, you mind telling me the IP? (grep SWCONTACT testssl.sh)?

**Comment by andrewbonney** [comment]
I'm afraid not, it's just an internal testing system, but I can provide further details to that address if useful, such as the full output of the 'openssl s_client -status' request.

**Comment by dcooper16** [comment]
Hi @andrewbonney ,

The `--phone-out` option isn't relevant here:
```
--phone-out    allow to contact external servers for CRL download and querying OCSP responder
```
Suppose that the TLS server you are testing is tls.example.com and the OCSP responder is ocsp.example.com. If the `--phone-out` option is not provided, then testssl.sh will not contact ocsp.example.com, but testssl.sh will ask tls.example.com for status information.

This is something we haven't seen before. When asking for a stapled OCSP response, servers usually either provide no status information at all or they provide a response indicating that the server's certificate is not revoked. I also tested it on a local server I set up in which the server's certificate was revoked. We've never before heard of a case in which the TLS server responded with an error response from the OCSP server.

I was, however, able to set up a test that repeated your results. I used OpenSSL's s_server to create a TLS server, and it seems that when asked for status information, s_server just forwards whatever response it gets from the OCSP responder without checking the response.

It seems that testssl.sh can handle a case in which it queries the OCSP responder itself (because the `--phone-out` option was provided) and gets an error response, but not a case in which a TLS server staples an error response. We'll have to add some code to testssl.sh to see if the stapled OCSP response is actually an error response.
